methods/espresso/5.2.1.yml

```diff
@@ -1,16 +1,19 @@
+# Conjugate gradient, tetrahedron method and other smearing types cannot be rendered
+# to the input file yet and are hence deactivated
 Default:
   pw.x:
     pw_scf:
-      - path: /qm/wf/undefined/pw/undefined
-      - regex: '\/qm\/wf\/undefined\/psp\/.*'
-      - regex: '\/qm\/wf\/undefined\/smearing\/.*'
-      - regex: '\/qm\/wf\/undefined\/tetrahedron\/.*'
-      - path: /opt/diff/ordern/cg/undefined
-      - path: /linalg/diag/undefined/davidson/undefined
+      - path: /qm/wf/none/pw/none
+      - path: /qm/wf/none/smearing/gaussian
+      - regex: \/qm\/wf\/none\/psp\/.*
+#      - regex: \/qm\/wf\/none\/smearing\/.*
+#      - regex: \/qm\/wf\/none\/tetrahedron\/.*
+#      - path: /opt/diff/ordern/cg/none
+      - path: /linalg/diag/none/davidson/none
     pw_scf_bands_hse:
-      - path: /qm/wf/undefined/pw/undefined
-      - regex: '\/qm\/wf\/undefined\/psp\/.*'
-      - regex: '\/qm\/wf\/undefined\/smearing\/.*'
-      - regex: '\/qm\/wf\/undefined\/tetrahedron\/.*'
-      - path: /opt/diff/ordern/cg/undefined
-      - path: /linalg/diag/undefined/davidson/undefined
+      - path: /qm/wf/none/pw/none
+      - regex: \/qm\/wf\/none\/psp\/.*
+#      - regex: \/qm\/wf\/none\/smearing\/.*
+#      - regex: \/qm\/wf\/none\/tetrahedron\/.*
```

Comment:
Let's leave a comment to consider adding a list of possibilities instead of a '*' in case the full list is changed without having this application support a particular item